json_schemas/manifest.json

```diff
@@ -23,9 +23,9 @@
             "pull_url": {"type": "string", "maxLength": 511, "format": "https-url"},
             "channelback_url": {"type": "string", "maxLength": 511, "format": "https-url"},
             "clickthrough_url": {"type": "string", "maxLength": 511, "format": "https-url"},
-            "healthcheck_url": {"type": "string", "maxLength": 511},
-            "about_url": {"type": "string", "maxLength": 511},
-            "dashboard_url": {"type": "string", "maxLength": 511},
+            "healthcheck_url": {"type": "string", "maxLength": 511, "format": "https-url"},
```

Comment:
Our intent is to always extend these URLs based on the "base" URL of the manifest itself: https://github.com/zendesk/zendesk_channels/blob/master/app/models/channels/any_channel/registered_integration_service.rb#L148, so it seems like there may be a bug in that code?

Reply:
@smedberg, not sure what the bug is that you're referring to here?